functions/Get-DbaAvailabilityGroup.ps1

```diff
@@ -90,7 +90,7 @@ function Get-DbaAvailabilityGroup {
 				if ($IsPrimary) {
 					$defaults = 'ComputerName','InstanceName','SqlInstance','Name as AvailabilityGroup','IsPrimary'
 					$value = $false
-					if ($serverName -eq $ag.PrimaryReplicaServerName) {
+					if ($ag.PrimaryReplicaServerName -eq $serverName) {
```

Comment:
This makes no difference which side of the operator you put the values to be compared.

Do you have screenshots of it failing before this change and working after?